transforms/add-layout-property/__testfixtures__/basic.output.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
```

Comment:
Should we ignore these fixture files from eslint, because they are failing in the build?

Reply:
Yes, ignored it.